newfeature.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-use-before-define */
+/* eslint-disable max-classes-per-file */
+
+const Title = document.querySelector('.title');
+const author = document.querySelector('.author');
+const registeredBooks = document.querySelector('.book-list');
+const addButton = document.querySelector('.addbook');
+
+let Books = [];
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+// BookScreen class
+
+class BookScreen {
+  static addBooks() {
+    registeredBooks.innerHTML = '';
+    for (let i = 0; i < Books.length; i += 1) {
+      registeredBooks.innerHTML += `
+      <div class="title-and-author"> 
+        <p class="Title">"${Books[i].title}" by  ${Books[i].author}</p>
+        <button class="button" onclick="BookScreen.remove(${i})">remove</button>
+      </div>
```

Comment:
- [ ] Please kindly add CSS styles to the application to make it match the given wireframe. 👍 

## Your Design 
![book](https://user-images.githubusercontent.com/103386183/204769155-2826225f-7a9d-4da9-8257-ef4cb4228650.png)